guidance/models/_model.py

```diff
@@ -382,11 +382,9 @@ def next(self, logits):
                     # if we don't have a cached match flag compute it using the grammar
                     if next_node.match_version < self._token_trie.match_version:
                         next_byte_mask = self._parser.next_byte_mask()
-                        for (
-                            byte
-                        ) in (
-                            node.keys()
-                        ):  # we update all the children since the parser knows the full mask
+
+                        # we update all the children since the parser knows the full mask
+                        for byte in node.keys():  
```

Comment:
some of these changes are just undoing particularly egregious dissections that `black` did...e.g. it's wild that this loop iterator was split into 5 lines!

Reply:
realizing now that it makes this particular PR much harder to parse though, apologies for that!

Reply:
I'm guessing that that was done by the long trailing comment. That seems to be something that `black` is not overly keen on

Reply:
Did you just shift the comment and re-run `black` ?

Reply:
Yeah I ended up manually moving trailing comments up a line, but it'd be good to see if there's an automatic way to make black handle it.